### PR TITLE
Add namespaces to System.Diagnostics.FleVersionInfo.Tests

### PR DESCRIPTION
**Pull request by hughbe**
- Part of #2898
- Has no functional changes

### CODE REVIEW

**Comment by stephentoub** [comment]
LGTM.  Thanks, @hughbe.

File: src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/Interop/Interop.Windows.cs [hunk 1]
```diff
@@ -5,11 +5,14 @@
 using System.Runtime.InteropServices;
 using System.Text;
 
-internal partial class Interop
+namespace System.Diagnostics.Tests
 {
-    internal partial class mincore
+    internal partial class Interop
```
Comment (stephentoub) [inline src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/Interop/Interop.Windows.cs]: Actually, I missed this... this should not be put into a namespace.  Please revert this file.

Comment (hughbe) [inline src/System.Diagnostics.FileVersionInfo/tests/System.Diagnostics.FileVersionInfo.Tests/Interop/Interop.Windows.cs]: Fixed

**Comment by stephentoub** [comment]
LGTM.  Thanks.

**Pull request merged**